### ISSUE TITLE
Features: remove index tip box and hyperlinks to it

### ISSUE THREAD

**Issue by Singa-pirate** [AY2324S1-CS2103T-T08-2/CS2101#4]
Reason: for 2101, showing exact valid range is not helpful and may be penalised

**Comment by awhb** [comment]
You mean the index tip box at the start of the "features" section?

That's valuable imo, see feedback below from Amy previously:
````
"Optional fields" can link to a different section where the constraints/details are explained
````

Or did you mean something else?

**Comment by Singa-pirate** [comment]
My reason for removing that box is because it contains "2147483647" which is discouraged by 2101. The rest of the content in the box is optional for 2101 imo. We can still keep it if there's a less technical way to rephrase the input constraint

**Comment by awhb** [comment]
I think can just remove the exact valid range sentence and all the callbacks to valid range.

**Comment by Singa-pirate** [comment]
Okay that works too

**Comment by Singa-pirate** [comment]
Solved in commit 544b823.
